xbmc/addons/AddonDatabase.cpp

```diff
@@ -240,8 +242,15 @@ void CAddonDatabase::SyncInstalled(const std::set<std::string>& ids, const std::
     std::string now = CDateTime::GetCurrentDateTime().GetAsDBDateTime();
     BeginTransaction();
     for (const auto& id : added)
+    {
+      int enable = 0;
+
+      if (system.find(id) != system.end() || optional.find(id) != optional.end())
```

Comment:
system add-ons are enabled below, so should this be just `if (optional.find(id) != optional.end())`?